packages/mdc-list/foundation.ts

```diff
@@ -108,7 +108,7 @@ export class MDCListFoundation extends MDCFoundation<MDCListAdapter> {
     }
   }
 
-  /** @return The index of the item that was last focused. */
+  /** @returns The index of the item that was last focused. */
```

Comment:
Let's just make this "Returns the index of the item that was last focused."

Reply:
Done.